### PR TITLE
Attempt to get travis tests working

### PR DESCRIPTION
**Pull request by drpatelh**
* Rename `uppmax-devel` to `uppmax_devel` everywhere 
* Add `SCRATCH` variable to `.travis.yml` tests
* Add in `czbiohub` in `README.md`
* Alphabetically order everything

### CODE REVIEW

**Comment by drpatelh** [comment]
To summarise, `uppmax_devel` is failing because I have changed `uppmax-devel.config` to `uppmax_devel.config` which wont be currently pullable from the `master` branch until we merge this PR.

`czbiohub_aws` and `czbiohub_aws_highpriority` are failing because the settings in those configs try and access a bucket that only CZ BioHub employees can access.

File: .travis.yml [hunk 1]
```diff
@@ -17,13 +17,14 @@ install:
...
+  - NXF_VER='18.10.1' SCRATCH='~' # Specify a minimum NF version that should be tested and work. Set SCRATCH for prince.config.
+  - NXF_VER='' SCRATCH='~'        # Plus: get the latest NF version and check, that it works. Set SCRATCH for prince.config.
 
 script:
   # Run the pipeline with the test profile and test remote config
   - |
       grep "{.*includeConfig.*[a-z]*\.config\"" ${TRAVIS_BUILD_DIR}/nfcore_custom.config | \
       tr -s ' ' | \
       cut -d " " -f 2 | \
+      grep -v "czbiohub_aws" | \
```
Comment (drpatelh) [inline .travis.yml]: I have instructed Travis to ignore `czbiohub_aws` until we can find a permanent fix. 

This PR is ready to merge now. Hopefully, the tests should be passing for `uppmax_devel` after the merge as mentioned in previous comments.

**Comment by drpatelh** [comment]
@nf-core/core PPPPPPPPPPPPIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIINNNNNNNNNNNNNNNNNGGGGGGGGGGGGGGG

**Comment by MaxUlysse** [comment]
I believe there's still some issues with the tests

**Comment by drpatelh** [comment]
Yes, they are expected to fail until this is merged because the tests are trying to pull `uppmax_devel.config` on `master` which doesnt exist yet!

**Comment by drpatelh** [comment]
I havent made any controversial changes anyway...all of the configs are still the same. Want to create a dummy PR to see if things are working after the merge. Ok. Going to merge :running_man:

**Pull request merged**